safe/definitions/field_groups.py

```diff
@@ -66,6 +66,7 @@
         adult_ratio_field,
         elderly_ratio_field
     ],
+    'exclusive': False,
```

Comment:
We should probably annotate these with a comment - I think in 6 months time we might forget what it is for - or a new user may not intuitively understand its purpose.